addons/web/static/src/views/fields/properties/property_tags.js

```diff
@@ -212,10 +212,11 @@ export class PropertyTags extends Component {
         }
 
         // cycle trough colors
-        const tagColor =
+        let tagColor =
             this.props.tags && this.props.tags.length
                 ? (this.props.tags[this.props.tags.length - 1][2] + 1) % ColorList.COLORS.length
                 : parseInt(Math.random() * ColorList.COLORS.length);
+        tagColor = tagColor || 1;  // never select white by default
```

Comment:
white color is more like "no color" (to not show the tag in the kanban view)
 currently the color are a big buggy in dark mode because if the "milk refactor", but yes that have a neutral color :)